src/components/Dialog/Confirm/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import ConfirmDialog, { confirmDialogDataTestIds } from '.';
+import { dialogDataTestIds } from '..';
+
+describe('ConfirmDialog', () => {
+  const emptyCallback = () => {
+    // Do nothing
+  };
+
+  describe('given the confirm dialog is opened', () => {
+    it('renders a confirm dialog', () => {
```

Comment:
Should we test the content rendered as well?

Reply:
@hoangmirs added in [7716683](https://github.com/manh-t/react-survey/pull/88/commits/7716683de5b6720fb6059e948fd25a6f2c2bcb8e)